Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -228,18 +228,20 @@ function _flush(rootNode: AnimatedValue): void {
   animatedStyles.forEach(animatedStyle => animatedStyle.update());
 }
 
-type TimingAnimationConfig =  AnimationConfig & {
+type TimingAnimationConfig = AnimationConfig & {
   toValue: number | AnimatedValue | {x: number, y: number} | AnimatedValueXY,
   easing?: (value: number) => number,
   duration?: number,
   delay?: number,
+  numLoops?: number,
```

Comment:
Could this prop be added to `AnimationConfig` type and the `Animation` base class instead of in all classes?